slam-ws/src/nav_goal/bin/movebase_client

```diff
@@ -5,17 +5,16 @@ import rospy
 import sys
 
 if __name__ == '__main__':
-   if len(sys.argv) < 4:
-        rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
-   else:
-	try:
-		x = float(sys.argv[1])
-		y = float(sys.argv[2])
-		yaw = float(sys.argv[3])
+   	if len(sys.argv) < 4:
+		   rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
```

Comment:
Running `movebase_client` w/o any arguments doesn't log this to terminal, instead I get:
```
erick@erick-MacBookPro:~/Documents/Github/SLAM/slam-ws$ rosrun nav_goal movebase_client 
No handlers could be found for logger "rosout"
```

Reply:
huh, very strange. I'll look into it

Reply:
Fixed